src/commands/random.ts

```diff
@@ -0,0 +1,27 @@
+import * as Discord from 'discord.js';
+const urban = require('relevant-urban');
+
+module.exports = {
+    name: 'random',
+    description: "random",
+    async execute(message, args) {
+
+        let result = await urban(args).catch(e => {
+            return;
```

Comment:
If I'd be you - I'd re-execute the function (instead of just `return;`-ing), until it finds a word (or runs out of words to test), that is found inside the Urban Dictionary! 
It's going to require a bit of thinking on how to do it, but that would mean that this __only__ wouldn't fire, if there is no word in the message, which is inside the Urban Dictionary!